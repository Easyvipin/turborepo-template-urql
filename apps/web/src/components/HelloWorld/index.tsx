import React from "react";

interface IHelloWorldProps {}

const HelloWorld: React.FunctionComponent<IHelloWorldProps> = (props) => {
  return <div>HelloWorld</div>;
};

export default HelloWorld;
