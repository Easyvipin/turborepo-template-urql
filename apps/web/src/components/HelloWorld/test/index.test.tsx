import React from "react";
import { render } from "@testing-library/react";
import HelloWorld from "../index";

test("renders HelloWorld component correctly", () => {
  const { container } = render(<HelloWorld />);
  expect(container.firstChild).toMatchSnapshot();
});
