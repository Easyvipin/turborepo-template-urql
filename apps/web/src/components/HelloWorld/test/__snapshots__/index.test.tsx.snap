// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HelloWorld component correctly 1`] = `
<div>
  HelloWorld
</div>
`;
